node-graph/gpath-bool/src/lib.rs

```diff
@@ -318,6 +318,7 @@ fn flatten_vector(graphic_table: &Table<Graphic>) -> Table<Vector> {
 						}
 					})
 					.collect::<Vec<_>>(),
+				Graphic::Typography(_) => Vec::new(),
```

Comment:
you did even create a placeholder function which does not do anything, couldn't you use that here?